airflow/example_dags/subdags/subdag.py

```diff
@@ -0,0 +1,19 @@
+from airflow.models import DAG
+from airflow.operators import DummyOperator
+
+
+def subdag(name, args):
+    dag_subdag = DAG(
+      dag_id='example_subdag_operator.%s' % name,
```

Comment:
- `example_subdag_operator` should not be hard-coded, but should be a parameter passed to an argument named `parent_dag_name` 
- The `name` argument should be renamed to `child_dag_name` 

The function then becomes `def subdag(parent_dag_name, child_dag_name, args)`